README.md

```diff
@@ -4,19 +4,35 @@
 
 > This a web app built on Rails API as backend and React framework as a frontend. This app is used for reserving a doctor's appointment by patients. Users can create, view, edit and delete a list of appointment(s).
 
-## Demo Video
+## Live Demo Link
```

Comment:
- [ ] This live demo link does not work.  

![image](https://user-images.githubusercontent.com/57421823/199852974-8f41d704-bc6a-41f5-806c-2b56a654ec35.png)

Kindly fix your deployment so your users can see your app without the need to install it locally 👍